src/ui/pages/Settings/Settings.tsx

```diff
@@ -104,9 +127,10 @@ function SettingsMain() {
         <Frame>
           <VStack gap={0}>
             <FrameListItemAnchor
-              href="http://app.zerion.io/rewards"
+              href={`https://app.zerion.io/rewards&${addWalletParams}`}
```

Comment:
`${ZERION_ORIGIN}/rewards?${addWalletParams}`

The main thing is to change & to ? before addWalletParams

Reply:
✅